R/vulnerability/disasters-emergencies/england/build-index.R

```diff
@@ -0,0 +1,37 @@
+# ---- Load libraries and Functions ----
+library(tidyverse)
+library(demographr)
+
+source("R/utils.R")
+
+# ---- Load indicators ----
+indicators <-
+  load_indicators(
+    path = "data/vulnerability/disasters-emergencies/england",
+    key = "lad_code"
+  )
+
+# Check entry for all 2021 LAD codes
+lads_21 <- lookup_lad_counties_21 |>
+  filter(str_detect(lad_code, "^E")) |>
+  distinct(lad_code) 
+
+lads_21 |>
+  anti_join(indicators, by = "lad_code")
+
+indicators |>
+  anti_join(lads_21, by = "lad_code")
+
+indicators |>
+  dplyr::filter(if_any(everything(), ~is.na(.x))) 
+# City of London & Isle of Scilly are missing disability data
+# Deal with this with 'ignore_nas' argument below
+
+# ---- Build Index ----
```

Comment:
Do you not need to align indicators before calculating the domain score (by multiplying by negative one)? In your variable `indicators`, a low digital vulnerability score = high vulnerability where as for all the other variables, high score = high vulnerability.